kornia/color/hsv.py

```diff
@@ -60,9 +60,9 @@ def hsv_to_rgb(image: torch.Tensor) -> torch.Tensor:
 
     hi: torch.Tensor = torch.floor(h * 6) % 6
     f: torch.Tensor = ((h * 6) % 6) - hi
-    p: torch.Tensor = v * (torch.tensor(1.) - s)
-    q: torch.Tensor = v * (torch.tensor(1.) - f * s)
-    t: torch.Tensor = v * (torch.tensor(1.) - (torch.tensor(1.) - f) * s)
+    p: torch.Tensor = v * (torch.tensor(1.).to(image.device) - s)
+    q: torch.Tensor = v * (torch.tensor(1.).to(image.device) - f * s)
+    t: torch.Tensor = v * (torch.tensor(1.).to(image.device) - (torch.tensor(1.).to(image.device) - f) * s)
 
```

Comment:
its better to create a tensor just once so we dont have to move to gpu all the time something like:

```
one = torch.tensor(1.).to(image.device)
 p: torch.Tensor = v * ( one - s)
 q: torch.Tensor = v * (one - f * s)
 t: torch.Tensor = v * (one  - (one - f) * s)
```